### ISSUE TITLE
deploy to shinyapp.io with my results to share w/ collaborators

### ISSUE THREAD

**Issue by jgmeyerucsd** [davidaknowles/leafcutter#86]
I really like the interactive shiny app for the data. It would be most useful if I could share this with my non-computational collaborators.  

I tried pushing the image to shinyapps.io using:

``` R
library(rsconnect)
rsconnect:deployApp()
```
and it worked, but it doesn't carry my analysis results with it, and it tries to load for a second before crashing. 

Can you help me share the shiny visualization with my collaborators either this way or another way?

**Comment by jackhump** [comment]
Hi, thanks for the compliment. 
To get the shiny app to work with shinyapps.io, I included an expression at the start of the server.R script - lines 68-77.
```
if (!exists("introns")){
  load("example/Brain_vs_Heart_results.Rdata")
  defaultValue <- 12 #RBFOX1
  # for testing - simulate data aligned to genome without "chr" in chr name
  #introns_to_plot$chr <- gsub("chr","", introns_to_plot$chr)
  #row.names(counts) <- gsub("chr", "", row.names(counts))

}else{
  defaultValue <- NULL
}
```

So in theory if you change "example/Brain_vs_Heart_results.Rdata" to your own Rdata file and upload that with rsconnect it should then load your data in to the app. I wrote this about 18 months ago so I'm a little rusty on the specifics but that should work. Please let me know if it does and I can add something to the README.

the rsconnect::showLogs() function is really handy for telling you why the app is crashing. If it still doesn't work then post them here and I'll take a look.

**Comment by jgmeyerucsd** [comment]
Thanks for the help. I got it to work similar to how you suggested, except the. rdata you want shipped with the app must be in the same directory as the server.R and other shiny files: leafcutter/leafviz/ 

Then I changed the 'load(data)' line you had commented out the path to that .Rdata file without full path, started an R session in the same directory, and ran the commands I had in my original post. It worked!

BTW I also got this working on the windows subsystem for Linux (WSL) and recorded my steps in a wiki page format - I'll create a pull request next weekend to add to your wiki.
